src/dags/airflow_db_permissions.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 from airflow import DAG
-from common import OTAP_ENVIRONMENT, SLACK_ICON_START, MessageOperator, default_args, slack_webhook_token
+from common import  MessageOperator, default_args
```

Comment:
You're going to delete the extra space before MessageOperator at some time, why not now? 😉

Reply:
Ah yes I will, nicely spotted